src/content/docs/tips/astro-font.mdx

```diff
@@ -0,0 +1,160 @@
+---
+title: Setup Astro-Font
+description: Quickly configure Astro-Font for your Astro project
+astroRange: ^4.0.0
+---
+
+import { Tabs, TabItem, Steps } from "@astrojs/starlight/components";
+
+Install `astro-font` package using your preferred package manager:
+
+<Tabs>
+    <TabItem label="npm">
+        ```sh
+        npm install astro-font
+        ```
+    </TabItem>
+    <TabItem label="pnpm">
+        ```sh
+        pnpm add astro-font
+        ```
+    </TabItem>
+    <TabItem label="yarn">
+        ```sh
+        yarn add astro-font
+        ```
+    </TabItem>
+</Tabs>
+
+## With Local Fonts
+
+<Steps>
+1. Make sure you have the local font in `public/fonts` directory. This example will use IBM Plex Sans
+  
+2. Configure `astro-font` in `layout.astro`
+
+   ```astro title="layout.astro"
+   ---
+   import { AstroFont } from "astro-font";
+   ---
+   <AstroFont
+      config={[
+        {
```

Comment:
Shouldn't this be `src`? Haven't checked the docs

Reply:
src is mentioned couple of lines below. now sure what you meant 😕